pennylane/templates/subroutines/hilbert_schmidt.py

```diff
@@ -162,7 +162,7 @@ def compute_decomposition(
 
         # Unitary V conjugate
         for op_v in v_tape.operations:
-            decomp_ops.append(op_v.adjoint())
+            decomp_ops.append(qml.adjoint(qml.apply, lazy=False)(op_v))
```

Comment:
Can you comment, why is the apply function transformed?

Reply:
Oh, this change was made before I added support for giving `qml.adjoint` a single op.

```suggestion
            decomp_ops.append(qml.adjoint(op_v, lazy=False))
```